modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2639,6 +2639,7 @@ be-specific-and-imagine-you-are-asking-a-question-to-another-person=Be specific
 be-sure-to-browse-our-guestbook-for-the-latest-entries=Be sure to {0}browse{1} our guestbook for the latest entries!
 be-yourself-again=Be yourself again.
 beanshell=Beanshell
+because-the-data-does-not-persist-in-Liferay=Because the data does not persist in Liferay's database, some limitations are placed on proxy objects. Learn more.
```

Comment:
Hi @DavysonMelo, the `LearnResourcesContext` will be responsable to add the `Learn more` sentense. Right? So I think we need to remove it from here.

Reply:
My bad, I'll remove it